### ISSUE TITLE
Feature Request: Able to use Proxy

### ISSUE THREAD

**Issue by vsatmydynipnet** [8go/matrix-commander#22]
I patched my local setup the way below, but a parameter would be cool to be able to use matrix-commander in company networks. Thank you for this cool peace of software:
```
diff -u matrix-commander.py.ORIG matrix-commander.py.01.w.proxy.SAV 
--- matrix-commander.py.ORIG	2021-03-12 15:15:23.747356000 +0100
+++ matrix-commander.py.01.w.proxy.SAV	2021-03-12 15:20:19.495094000 +0100
@@ -2424,6 +2424,7 @@
         # Initialize the matrix client
         client = AsyncClient(
             homeserver, user_id, store_path=store_dir, config=client_config,
+            proxy="http://10.50.50.3:8118",
         )
 
         pw = getpass.getpass()
@@ -2497,6 +2498,7 @@
         device_id=credentials["device_id"],
         store_path=store_dir,
         config=client_config,
+        proxy="http://10.50.50.3:8118",
     )
     client.restore_login(
         user_id=credentials["user_id"],
```

**Comment by 8go** [comment]
Thank you @vsatmydynipnet for suggesting this. Thanks @MacLemon for the :+1: 

I implemented it today. 
Have a look at bc53cac7c0dc1f34281773a8fbe667e082711a6c

@vsatmydynipnet : Please get today's version, test it and provide feedback to verify that it is working for you.

**Comment by 8go** [comment]
I assume the fix in bc53cac is working for @vsatmydynipnet 

Hence closing this issue, feature request completed.
